### PR TITLE
Task/style consistency cleanup

### LINKED ISSUE #75: Apply EsLint Rules Prior to Deliver

### PR DESCRIPTION
**Pull request by bergsalex**
Closes #75

### CODE REVIEW

File: libs/ngx-genome-karyotype-plots/src/lib/genome-karyotype-plot/genome-karyotype-plot.component.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <ng-content select="[slot=header]"></ng-content>
 <div class="row" id="hapkaryoplot">
-  <haploqa-modules-svg-element
-    [name]="name" [width]="width"
-    [height]="height" [margin]="margin"
-    (svgClick)="handleClick($event)">
+  <haploqa-modules-svg-element [name]="name"
```
Comment (anna-lamoureux) [inline libs/ngx-genome-karyotype-plots/src/lib/genome-karyotype-plot/genome-karyotype-plot.component.html:3]: I've tried this style of indentation and didn't particularly care for it BUT if you're committed to it and/or your formatter is configured for it, I don't see a reason why it shouldn't be fine here

File: libs/ngx-genome-karyotype-plots/src/lib/svg-tools.service.ts [hunk 2]
```diff
@@ -34,23 +33,21 @@ export interface SnpBinItem {
...
-  public axesGroup: any;
-  public chrOrdinalScale: any;
-  public genomeScale: any;
-  public legend!: any;
+  public axesGroup!: D3Selection;
+  public chrOrdinalScale!: any;
+  public genomeScale!: d3.ScaleLinear<number, number, never>;
+  public legend!: D3Selection;
 
   private xAxisSize = 100;
```
Comment (anna-lamoureux) [inline libs/ngx-genome-karyotype-plots/src/lib/svg-tools.service.ts:41]: Should these two also have underscores?

File: libs/ngx-genome-karyotype-plots/src/lib/genome-karyotype-plot/genome-karyotype-plot.component.html (hunk 1 above)
Comment (bergsalex) [reply libs/ngx-genome-karyotype-plots/src/lib/genome-karyotype-plot/genome-karyotype-plot.component.html:3]: Not committed to it, but this is just an example.

File: libs/ngx-genome-karyotype-plots/src/lib/svg-tools.service.ts [hunk 3]
```diff
@@ -258,6 +247,7 @@ export class SvgToolsService {
                   this.strainSvc.strainColor(currStrain1))
               })
               .on("mouseout", () => this.clearHaplotypeHightlight());
+            // eslint-disable-next-line no-prototype-builtins
             if(haplotypeMap.hasOwnProperty(currStrain1)) {
```
Comment (anna-lamoureux) [inline libs/ngx-genome-karyotype-plots/src/lib/svg-tools.service.ts:251]: In one of my repos that kept calling this issue out, the solution I found was to do like `Object.prototype.hasOwnProperty.call(objectToCheck, key)`. It's longer but it prevents you from needing eslint-disable lines on each one

**Review by anna-lamoureux — APPROVED** [review]

**Pull request merged**